### ISSUE TITLE
Passing empty string for parameter --reject makes ncu to detect empty package.json

### ISSUE THREAD

**Issue by jonArzelus** [raineorshine/npm-check-updates#1201]
I have detected some problem with the current way ncu handles the `--reject` parameter when an empty string (`""`) is provided for this parameter; Currently it checks the dependencies on the home directory of the current user (or somewhere else that is not the current directory). (even if you configure `--cwd` correctly), I have checked the `--loglevel silly` for correct detection of the `package.json` intended to upgrade to be correct.

Steps to reproduce:

Create a new project (I am using yarn package manager but npm also will work) and add a outdated dependency for later checking with ncu:
```mkdir ~/project
cd ~/project
yarn init
yarn add inherits@1.0.0
ncu --cwd ~/project --reject "" --loglevel silly
```

## Current Behavior
It outputs the following:
```Initializing
Running in local mode
Finding package file data
Checking /home/user/Documents/project/package.json

Options:
{
  args: [],
  cache: false,
  cacheExpiration: 10,
  cacheFile: '~/.ncu-cache.json',
  cacher: undefined,
  cli: true,
  concurrency: 8,
  cwd: '/home/user/project',
  deep: false,
  dep: 'prod,dev,bundle,optional',
  errorLevel: 1,
  format: [],
  json: false,
  loglevel: 'silly',
  minimal: false,
  packageManager: 'npm',
  prefix: undefined,
  reject: [ '', '' ],
  retry: 3,
  target: 'latest',
  workspace: []
}

Current versions:
{}

Fetching latest versions

Fetched versions:
{}

Upgraded versions:
{}

No dependencies.
```

## Expected Behavior
It should output the following:
```Initializing
Running in local mode
Finding package file data
Checking /home/user/Documents/project/package.json

Options:
{
  args: [],
  cache: false,
  cacheExpiration: 10,
  cacheFile: '~/.ncu-cache.json',
  cacher: undefined,
  cli: true,
  concurrency: 8,
  cwd: '/home/user/project',
  deep: false,
  dep: 'prod,dev,bundle,optional',
  errorLevel: 1,
  format: [],
  json: false,
  loglevel: 'silly',
  minimal: false,
  packageManager: 'npm',
  prefix: undefined,
  reject: [],
  retry: 3,
  target: 'latest',
  workspace: []
}

Current versions:
{ inherits: '1.0.0' }

Fetching latest versions
[====================] 1/1 100%

Fetched versions:
{ inherits: '2.0.4' }

Upgraded versions:
{ inherits: '2.0.4' }

 inherits  1.0.0  →  2.0.4

Run ncu --cwd /home/user/project --loglevel silly -u to upgrade package.json
```
OR, at least, it should warn you about the empty `--reject` parameter and work this way (or warn about this behaviour somewhere in the docs, idk). One thing is for sure, the current behaviour looks faulty to me.

Thanks!

 ---

- node version: v16.13.0
- npm version: 8.19.2
- npm-check-updates version: 16.3.8
- output from npm -g ls --depth=0: 
```├── babel-eslint@10.1.0
├── corepack@0.10.0
├── eslint-config-react-app@7.0.1
├── eslint@8.20.0
├── npm-check-updates@16.3.8
├── npm@8.19.2
├── pm2@5.1.2
├── prettier@2.6.2
├── serve@14.0.1
├── source-map-explorer@2.5.2
└── yarn@1.22.19
```

- [X] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [X] I am using the latest version of `npm-check-updates`
- [X] I am using `node >= 14`

**Comment by raineorshine** [comment]
Hi, thanks for reporting.

This turned out to be caused by a bug in which options got duplicated. For some reason the duplicated empty reject excluded all dependencies. My previous solution didn't cover all options. Now I have generalized it and added the necessary tests.

Fixed and published in `v16.3.9`

**Comment by jonArzelus** [comment]
Nice! Looking forward for it, thanks!
